src/TitleDetails/TitleDetails.js

```diff
@@ -156,38 +162,63 @@ const TitleDetails = ({
       lastMenu={lastMenu}
     >
       <Row end="xs">
-        <Col xs={12}>
+        <Col xs={10}>
+          <MessageBanner
+            show={isOrderClosed}
```

Comment:
I don't think we need to use this technique. We use `{isOrderClosed && <MessageBanner show />}` to render element only when it's needed